lisa/util/__init__.py

```diff
@@ -53,7 +53,7 @@
 
 
 # used to filter ansi escapes for better layout in log and other place
-__ansi_escape = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
+__ansi_escape = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~]|[=<>a-kzNM78])")
```

Comment:
What's the reason to change it? It may impact widely.

Reply:
It does not filter "\x1b=............\x1b"
Example:
Text: "\x1b[?1h\x1b=\rAdd linux-next specific files for 20230221\x1b[m\r\n\r\x1b[K\x1b[?1l\x1b>"
Filter Result: '\x1b=\rAdd linux-next specific files for 20230221\r\n\r\x1b>'
After change: '\rAdd linux-next specific files for 20230221\r\n\r'

Ref: https://stackoverflow.com/a/45448194

Reply:
It can be like below. And add above example, so it's easy to prevent regression next time.

\x1b(?:[@-Z\\-_=<>a-kzNM78]|\[[0-?]*[ -/]*[@-~])

Reply:
Did you mean the below?
"\x1B(?:[@-Z\\-_=<>a-kzNM78]|[[0-?]*[ -/]*[@-~])"

The regex "\x1b(?:[@-Z\-_=<>a-kzNM78]|[[0-?][ -/][@-~])" is not filtering correctly.

Reply:
oh, okay I understand, Github is removing double \ and * from the comments 😄